src/app/demos/default-styles/index.html

```diff
@@ -0,0 +1,107 @@
+  </h3>
+
+  <h4>
+    Heading 4
+  </h4>
+
+  <h5>
+    Heading 5
+  </h5>
+
+  <h6>
+    Heading 6
+  </h6>
+
+  <p>
+    Here, then, was this grey-headed, ungodly old man, chasing with curses a Job's whale round the world, at the head of a crew, too, chiefly made up of mongrel renegades, and castaways, and cannibals—morally enfeebled also, by the incompetence of mere unaided virtue or right-mindedness in Starbuck, the invunerable jollity of indifference and recklessness in Stubb, and the pervading mediocrity in Flask. Such a crew, so officered, seemed specially picked and packed by some infernal fatality to help him to his monomaniac revenge. How it was that they so aboundingly responded to the old man's ire—by what evil magic their
+  </p>
+
+  <p>
+    HSuddenly the foremost Martian lowered his tube and discharged a canister of the black gas at the ironclad. It hit her larboard side and glanced off in an inky jet that rolled away to seaward, an unfolding torrent of Black Smoke, from which the ironclad drove clear. To the watchers from the steamer, low in the water and with the sun in their eyes, it seemed as though she were already among the Martians. They saw the gaunt figures separating and rising out of the water as they retreated shoreward, and one of them raised the camera-like generator of.
+  </p>
+
+  <blockquote>
+    This is a blockquote.
+  </blockquote>
+
+  <table>
+    <tr>
+      <th>
+        Table Heading
+      </th>
+      <th>
+        Table Heading
+      </th>
+      <th>
+        Table Heading
+      </th>
+    </tr>
+    <tr>
+      <td>
+        Table Data
+      </td>
+      <td>
+        Table Data
+      </td>
+      <td>
+        Table Data
+      </td>
+    </tr>
+    <tr>
+      <td>
+        Table Data
+      </td>
+      <td>
+        Table Data
+      </td>
+      <td>
+        Table Data
+      </td>
+    </tr>
+    <tr>
+      <td>
+        Table Data
+      </td>
+      <td>
+        Table Data
+      </td>
+      <td>
+        Table Data
+      </td>
+    </tr>
+  </table>
+
+  <ul>
+    <li>Unordered List</li>
+    <li>Passepartout was enraged beyond expression by the unpropitious weather</li>
+    <li>Had the hour of adversity come? Passepartout was as much excited as if the twenty thousand pounds were to come from his own pocket</li>
+    <li>Passepartout was enraged beyond expression by the unpropitious weather. Everything had gone so well till now! Earth and sea had seemed to be at his master's service; steamers and railways obeyed him; wind and steam united to speed his journey. Had the hour of adversity come? Passepartout was as much excited as if the twenty thousand pounds were to come from his own pocket. The storm exasperated him, the gale made him furious, and he longed to lash the obstinate sea into obedience. Poor fellow! Fix carefully concealed from him his own satisfaction, for, had he betrayed it, Passepartout could scarcely have restrained himself from personal violence.</li>
```

Comment:
What sort of playfulness is this! Get outta here. This is a professional setting sir!